src/settings.cpp

```diff
@@ -0,0 +1,33 @@
+#include <cubos/settings.hpp>
+
+cubos::Settings* cubos::Settings::_instance = nullptr;

```

Comment:
Add `using namespace cubos;` and use auto for variable definitions to make it less verbose.